website/content/en/highlights/2022-03-22-0-21-0-upgrade-guide.md

```diff
@@ -28,6 +29,32 @@ We cover them below to help you upgrade quickly:
 
 ### Breaking changes
 
+#### Syntax changes for non-VRL paths {#path-syntax-changes}
+
+Previously, there were two different ways to describe paths. VRL (remap transforms) use a newer syntax, while everything
```

Comment:
nit: VRL is used in more places than the remap transform (any transform that has condition fields also uses VRL)

Reply:
I guess it's not a nit actually, we should call out that this change also doesn't impact component fields that are a condition.